elyses-enchantments/enchantments.js

```diff
@@ -48,8 +50,9 @@ export function insertItemAtTop(cards, newCard) {
  * @returns {number[]} the cards without the removed card
  */
 export function removeItem(cards, position) {
-  cards.splice(position, 1);
-  return cards;
+  const newCards = Array.from(cards);
+  newCards.splice(position, 1);
+  return newCards;
```

Comment:
Try to think of a way that does not mutate arrays.

```js
  return [ ...<items before removal>, ...<items after removal> ];
```

See [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice)